test/unit/org/apache/cassandra/cql3/validation/operations/SelectTest.java

```diff
@@ -34,13 +35,11 @@
 import static org.apache.cassandra.utils.ByteBufferUtil.EMPTY_BYTE_BUFFER;
 import static org.apache.cassandra.utils.ByteBufferUtil.bytes;
 
-
```

Comment:
These changes seem unrelated to the patch, maybe they are survivors from the previous version of the new tests.

Reply:
agree, best to revert to simplify diff.